src/main/kotlin/com/ilgijjan/integration/music/infrastructure/LyriaMusicGenerator.kt

```
@@ -81,19 +79,14 @@ class LyriaMusicGenerator(
 
-                return webClient.post()
-                    .bodyValue(requestBody)
+                return restClient.post()
+                    .body(requestBody)
                     .retrieve()
-                    .bodyToMono(Map::class.java)
-                    .block() ?: throw RuntimeException("Lyria 응답 없음")
+                    .body(Map::class.java) ?: throw RuntimeException("Lyria 응답 없음")
 
-            } catch (e: WebClientResponseException) {
+            } catch (e: HttpClientErrorException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)